liquidprompt

```diff
@@ -1912,14 +1961,21 @@ _lp_hostname_color() {
     fi
 
     if _lp_hostname; then
+
         case "$lp_connection" in
         lcl)
             lp_hostname_color+="@${LP_COLOR_HOST}${lp_hostname}${NO_COL}"
             ;;
         ssh)
+            local hostname=
+            if _lp_create_link "ssh://${lp_hostname}" "$lp_hostname"; then
```

Comment:
Thinking out loud; we can find the IP address that we connected to with
```bash
IFS=" " read -r client_ip client_port server_ip server_port <<<"$SSH_CONNECTION"
```
So then we could make the link `"ssh://${server_ip}:${server_port}"` instead, which is probably more likely to work?